isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -783,6 +784,785 @@ namespace Isis {
+            if (i == 0) {
+              parameterNamesList.append( str.arg(" RA  ").arg("dd") );
+              correctionUnitList.append("dd");
+            }
+            else {
+              switch(i) {
+
+              case 1:
+                parameterNamesList.append( str.arg("     ").arg("dd/s") );
+                correctionUnitList.append("dd");
+                break;
+              case 2:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+                correctionUnitList.append("dd/s^2");
+                break;
+              case 3:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+                correctionUnitList.append("dd/s^3");
+                break;
+              default:
+                  parameterNamesList.append(str.arg("     ").arg("dd/s^"+toString(i) ) );
+                  correctionUnitList.append("dd/s^"+toString(i));
+
+              }
+            }
+          }
+          for (int i = 0; i < nPointingCoefficients; i++) {
+            finalParameterValues.push_back(coefDEC[i] * RAD2DEG);
+            if (i == 0) {
+              parameterNamesList.append( str.arg("DEC  ").arg("dd") );
+              correctionUnitList.append("dd");
+
+            }
+            else {
+
+              switch(i) {
+
+              case 1:
+                parameterNamesList.append( str.arg("     ").arg("dd/s") );
+                correctionUnitList.append("dd/s");
+                break;
+              case 2:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+                correctionUnitList.append("dd/s^2");
+                break;
+              case 3:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+                correctionUnitList.append("dd/s^3");
+                break;
+              default:
+                  parameterNamesList.append(str.arg("     ").arg("dd/s^"+toString(i) ) );
+                  correctionUnitList.append("dd/s^"+toString(i) );
+
+              }
+            }
+          }
+          for (int i = 0; i < nPointingCoefficients; i++) {
+            finalParameterValues.push_back(coefTWI[i] * RAD2DEG);
+            if (i == 0) {
+              parameterNamesList.append( str.arg("TWI  ").arg("dd") );
+              correctionUnitList.append("dd");
+              }
+            else {
+              switch(i) {
+              case 1:
+                parameterNamesList.append( str.arg("     ").arg("dd/s") );
+                correctionUnitList.append("dd/s");
+                break;
+              case 2:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+                correctionUnitList.append("dd/s^2");
+                break;
+              case 3:
+                parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+                correctionUnitList.append("dd/s^3");
+                break;
```

Comment:
Again: the default should handle cases 2 and 3, so they should be removed.

Reply:
See comment above above above...oh never mind...I'll just stop by your office.